lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm

```diff
@@ -120,7 +122,8 @@ sub call_rpc {
                     if (my $err = $tx->error) {
                         $details .= ', code: ' . ($err->{code} // 'n/a') . ', response: ' . $err->{message};
                     }
-                    warn "WrongResponse [$msg_type], details: $details";
+                    $log->info("WrongResponse [$msg_type], details: $details");
+                    stats_inc("websocket.wrong_response", {tags => ["rpc:" . $req_storage->{name}]});
```

Comment:
We probably want a callback here so users of the code can supply their own metrics code.

Reply:
Since we already have `$rpc_failure_cb` to call shortly, that can work here in user code:

```perl
$app->plugin(web_socket_proxy => {
    ...,
    rpc_failure_cb => sub {
        my ($c, $res, $req_storage) = @_;
        stats_inc(..., {tags => [ ..., $req_storage->{name} ]});
        return undef;
    },
    ...
});